dev-client/src/components/common/Card.tsx

```diff
@@ -1,32 +1,45 @@
 import {Box} from 'native-base';
 import {IconButton} from './Icons';
 import {Pressable} from 'react-native';
+import {forwardRef} from 'react';
 
-export const CardCloseButton = ({onPress}: {onPress: () => void}) => {
+export const CardTopRightButton = forwardRef(
+  (props: React.ComponentProps<typeof IconButton>, ref) => {
+    return (
+      <Box position="absolute" top="0px" right="0px" p="8px">
+        <Pressable onPress={props.onPress}>
+          <IconButton ref={ref} {...props} />
+        </Pressable>
+      </Box>
+    );
+  },
+);
+
+export const CardCloseButton = (
+  props: Omit<React.ComponentProps<typeof CardTopRightButton>, 'name'>,
+) => {
   return (
-    <IconButton
+    <CardTopRightButton
       name="close"
       size="sm"
       background="grey.200"
       _icon={{color: 'action.active'}}
       borderRadius="full"
-      onPress={onPress}
+      {...props}
```

Comment:
mmm what i was trying to do is make the `IconButton` pressable outside its bounds, since there was an issue where sometimes hitting the close button on cards would hit the corner of the card and navigate to the card's item's screen. there is probably a more elegant way to do this but this is what i was able to make work!